torchao/prototype/low_bit_optim/subclass_fp8.py

```diff
@@ -88,12 +110,15 @@ def _(func, types, args, kwargs):
         assert dst.block_size == src.block_size
         dst.codes.copy_(src.codes)
         dst.scale.copy_(src.scale)
+        dst.k.copy_(src.k)
```

Comment:
Need to check if k is not None

Reply:
understood. since copy_ functionality if for tensor. it will lead to error if we implement it on. is my assumption correct? anything else i am missing?